src/components/application_manager/test/mobile_message_handler_test.cc

```diff
@@ -148,8 +148,8 @@ class MobileMessageHandlerTest : public testing::Test {
     // Arrange
     size_t payload_size = data.size();
     size_t full_data_size = data.size() + PROTOCOL_HEADER_V2_SIZE;
-    Message* message =
-        HandleIncomingMessage(protocol_version, data, payload_size);
+    const auto message = std::shared_ptr<Message>(
```

Comment:
@SNiukalov Can't see any sharing here. What is teh reason to create shared_ptr?

Reply:
@AByzhynar @LitvinenkoIra 
Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/41/commits/be4a738c2392d04c4fd0cd9a2dee39ce76b94c6e